src/redux/missionsSlice.js

```diff
@@ -0,0 +1,55 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+
+const url = 'https://api.spacexdata.com/v3/missions';
+
+export const fetchMissions = createAsyncThunk('missions/fetch', async () => {
+  const res = await fetch(url);
+  const data = await res.json();
+  return data.map((e) => ({
+    mission_id: e.mission_id,
+    mission_name: e.mission_name,
+    description: e.description,
+    reserved: false,
```

Comment:
- [ ] You have done a great job, just a minor detail here. Let's create a directory for the redux store as stated in the requirements to be consistent so it looks something like this` redux/rockets/rocketsSlice.js ` `redux/missions/missionsSlice.js`